flatgfa/src/main.rs

```diff
@@ -104,6 +106,23 @@ fn main() -> Result<(), &'static str> {
         Some(Command::Depth(_)) => {
             cmds::depth(&gfa);
         }
+        Some(Command::Chop(sub_args)) => {
+            let store = cmds::chop(&gfa, sub_args)?;
+            let flat = flatgfa::FlatGFA {
```

Comment:
Maybe also deserves a comment about how we'd like to change the encapsulation here someday: to summarize, the problem is that we're creating a "hybrid" `FlatGFA` that borrows from two different underlying stores, and we don't have a good (Rust-safe) way to express that.